### DISCUSSION TITLE
Correct way to put all generated relations in the same package

### DISCUSSION THREAD

**Discussion by kolemannix** [oyvindberg/typo#125] [Q&A]
Is it possible not to have a package per table? Here is my failed attempt to put everything in a `gen` package rather than have a package per table:

```scala
    override def fragments(source: Source): (sc.QIdent, String) = {
      def forRelPath(relPath: RelPath): (sc.QIdent, String) =
        (
          pkg,
          relPath.segments.last.replace(".sql", "")
        )

      source match {
        case relation: Source.Relation =>
          // Default impl that includes schema name
          // We do not want the schema name
          // (pkg / relation.name.schema.toList.map(pkgSafe), relation.name.name)


          // Attempt to put all generated classes in one package fails
          (pkg, "gen")
        case Source.SqlFile(relPath) => forRelPath(relPath)
      }
    }
```

It seems that typo is expecting a unique name as it only writes the very first relation it sees and names it `Gen`. I think I am wrongly setting the `name` here; and it is just not supported to do what I am trying to do, which is completely OK if so.

**Comment by oyvindberg** [comment]
You're very close!

the `String` is the base name for all generated classes. say something like `(pkg / sc.Ident("gen"), relation.name.name)` and I think it should work
